src/user/components/LoginForm/LoginForm.tsx

```diff
@@ -33,6 +33,10 @@ export const LoginForm = ({ handleFieldValueChange, handleSubmit, isSubmitDisabl
     inputRef.current?.focus();
   }, []);
 
+  useLayoutEffect(() => {
+    inputRef.current?.focus();
```

Comment:
It will change focus on every change of error state. Do we need to change focus when error is gone at the moment when user start changing password, for example?